package.json

```diff
@@ -40,7 +40,7 @@
     "build:deps": "bash ./scripts/dependencies.sh --doctor -u --doctorTest \"npm run test:deps\" --reject \"@patternfly/*, @redhat-cloud-services/frontend*, react-router, victory*\"",
     "build:deps-core": "bash ./scripts/dependencies.sh --doctor -u --doctorTest \"npm run test:deps\" --filter \"@patternfly/*, @redhat-cloud-services/frontend*, victory*\"",
     "build:docs": "run-s -l test:docs docs:md",
-    "build:ephemeral": "run-s -l build:pre build:js build:post test:integration-ephemeral",
+    "build:ephemeral": "run-s -l build:pre build:js build:post test:integration",
```

Comment:
Gotcha. thought so might would need some more changes. I am trying to understand the difference between the, why test passes for one and fails for other.

Looks similar commands yet, for  the ` integration-ephemeral ` code.test.js fails, I tried locally and it passes, maybe something to do with node or npm version, not sure . Any idea what might have caused it ?
```
    "test:integration": "jest ./tests",
    "test:integration-ephemeral": "TZ=UTC jest ./tests --no-cache --testPathIgnorePatterns ./tests/dist.test.js",
```

If adding new npm scripts/commands would solve the failure test, or get us past it, or ignore the failure with warning and not fail the job that would help. 

Or can we drop integration testing for ephemeral since, we have github actions pipeline already accomplishing those in other pr checks part. (Again, Not sure if these are used anywhere else in pipeline though)

I am looking for going past that integration test failure, so we can unblock prchecks, since the failure can be ignored safely as you suggested.

So, what would be your recommendation, should I replace  `tests:integration-ephemeral ` with  `tests:integration` wherever it's used. Since, I validated the EE spun from this, that would be stage-stable and that would work for prchecks, if I am not wrong. 

Or Feel free to close this PR, for a better way. I am mostly flying blind here, picking up as I move along.

Reply:
No worries! It sounds like the "ephemeral" part of the script name is what's causing part of the issue. 

To be more clear the script `test:integration-ephemeral` is meant to be run against the image/container build used on Quay and pushed towards the related environments, not necessarily a QE test environment. We just happened to name it "ephemeral" because that's the label that was given to us when it was created.

The add on of QE attempting to run PR checks against the already customized container/image NPM script is what is causing issues... because whatever we "bypassed" with that container image build already had issues.

So yes we can skip the "integration checks"... that gives us the next steps of...

1. Determine where `test:integration-ephemeral` is being called on the QE PR checks. If it's in this repo on something in this [deploy directory](https://github.com/RedHatInsights/curiosity-frontend/tree/main/deploy), , add it to this PR. If not adjust in where you need to and then we'll update this PR by...
1. Adding or pointing to an existing NPM script that ignores the `test:integration-ephemeral`. If it's related to a `build:` script **we can add a custom NPM script for QE ephemeral PR testing** like `build:qe: run-s -l build:pre build:js build:post` or something similar

I recognize it's frustrating, apologies, I think we're close!

Reply:
I added the `"build:pr_checks": "run-s -l build:pre build:js build:post"` , but for some reason the the PRchecks job won't pick up the ENV variable, any idea, what I might need to do to test this?